pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -483,6 +483,14 @@ void setBootnodes(final List<String> values) {
       arity = "1")
   private final Integer txPoolMaxSize = PendingTransactions.MAX_PENDING_TRANSACTIONS;
 
+  @Option(
+      names = {"--tx-pool-retention-hours"},
+      paramLabel = MANDATORY_INTEGER_FORMAT_HELP,
+      description =
+          "Maximum retention period of pending transactions, expressed in hours. Represented by primitive type 'long'. (default: ${DEFAULT-VALUE})",
```

Comment:
nit: drop the redundant word 'expressed' and user's don't care how we represent the value internally.
```suggestion
          "Maximum retention period of pending transactions in hours (default: ${DEFAULT-VALUE})",
```